packages/arm-core/src/components/DrawerOverlay/panels/LargeDrawerPanel.tsx

```diff
@@ -0,0 +1,19 @@
+import { breakpoint } from '@amsterdam/asc-ui'
+import styled from 'styled-components'
+import DrawerPanel from './DrawerPanel'
+
+const LargeDrawerPanel = styled(DrawerPanel)`
+  @media screen and ${breakpoint('min-width', 'tabletM')} {
+    width: 356px;
+  }
+
+  @media screen and ${breakpoint('min-width', 'laptop')} {
+    width: 596px;
+  }
+
+  @media screen and ${breakpoint('min-width', 'laptopL')} {
+    width: 756px;
```

Comment:
Where do these magic numbers come from?

Reply:
These are as specified by the design made by Iris, taking into account the width of the page. I wanted to include some sensible defaults. If a custom panel is needed one can use `styled(DrawerPanel)` for their specific use-cases.